notebooks/beginner.ipynb

```diff
@@ -614,19 +584,503 @@
    "source": [
     "k.expand.to_s"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SymEngine::RealDouble\n",
+    "\n",
+    "SymEngine::RealDouble can be constructed by converting any ruby Float into SymEngine"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "1.2"
+      ],
+      "text/plain": [
+       "#<SymEngine::RealDouble(1.2)>"
+      ]
```

Comment:
I agree, the one @zverok showed is much better than the current representation and should be improved. Thanks for the suggestion.